va/va.h

```diff
@@ -2547,6 +2549,23 @@ typedef struct _VAEncMiscParameterCustomRoundingControl
         uint32_t    value;
     }   rounding_offset_setting;
 } VAEncMiscParameterCustomRoundingControl;
+
```

Comment:
I assume it misses a written description of purpose of the structure and its usage.

Reply:
accept